Interfaces/GCClassic/main.F90

```diff
@@ -1706,6 +1724,35 @@ PROGRAM GEOS_Chem
              CALL Error_Stop( ErrMsg, ThisLoc )
           ENDIF
 
+          !------------------------------------------------------------------------
+          ! Set species concentration back to v/v dry air (mol/mol dry air)
+          ! prior to setting restart file arrays. Doing this conversion every
+          ! timestep is required for bit-for-bit reproducibility when breaking up
+          ! runs in time.
+          !------------------------------------------------------------------------
+          ! Turn off diagnostics timer
+          IF ( Input_Opt%useTimers ) THEN
+             CALL Timer_End( "Diagnostics", RC )
+          ENDIF
+
```

Comment:
@lizziel: Should we also start the unit conversion timer here?

Reply:
I don't think so since the unit conversion timer start and end are within Convert_Spc_Units.